### PR TITLE
[FLINK-7024][table]Add supported for selecting window proctime/rowtim…

### PR DESCRIPTION
**Pull request by sunjincheng121**
…e on row-based Tumble/Slide window

Thanks for contributing to Apache Flink. Before you open your pull request, please take the following check list into consideration.
If your changes take all of the items into account, feel free to open your pull request. For more information and/or questions please refer to the [How To Contribute guide](http://flink.apache.org/how-to-contribute.html).
In addition to going through the list, please provide a meaningful description of your changes.

- [x] General
  - The pull request references the related JIRA issue ("[FLINK-7024][table]Add supported for selecting window proctime/rowtime on row-based Tumble/Slide window")
  - The pull request addresses only one issue
  - Each commit in the PR has a meaningful commit message (including the JIRA id)

- [ ] Documentation
  - Documentation has been added for new functionality
  - Old documentation affected by the pull request has been updated
  - JavaDoc for public methods has been added

- [x] Tests & Build
  - Functionality added by the pull request is covered by tests
  - `mvn clean verify` has been executed successfully locally or a Travis build has passed

### CODE REVIEW

**Comment by sunjincheng121** [comment]
CC @wuchong @twalthr :-)

File: flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala [hunk 1]
```diff
@@ -654,17 +654,23 @@ case class WindowAggregate(
...
-        case SlidingGroupWindow(_, _, size, _) if isRowCountLiteral(size) =>
-          failValidation("Window start and Window end cannot be selected " +
-                           "for a row-count Sliding window.")
+            case SlidingGroupWindow(_, _, size, _) if isRowCountLiteral(size) =>
+              failValidation(
+                "Window start and Window end cannot be selected " +
+                  "for a row-count Sliding window.")
 
-        case _ => // ok
+            case _ => // ok
```
Comment (wuchong) [inline flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala]: Can you add a comment here that the `RowtimeAttribute` and `ProctimeAttribute` should pass ?

Comment (sunjincheng121) [inline flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/logical/operators.scala]: Sure. good idea.

**Comment by sunjincheng121** [comment]
@wuchong Thanks for your review. I had update the PR.

**Comment by sunjincheng121** [comment]
Hi @wuchong I had rebase the PR. Please look at the PR. :-) 

Thanks, jincheng

**Pull request closed without merging**